tests/unit_test/test_datasets.py

```diff
@@ -4,13 +4,17 @@
 def test_load_data():
     train_data = mz.datasets.wiki_qa.load_data('train', task='ranking')
     assert len(train_data) == 20360
+    train_data, _ = mz.datasets.wiki_qa.load_data('train', task='classification')
+    assert len(train_data) == 20360
```

Comment:
this line didn't test anything new

Reply:
This line is to test the dataset generation for `classification`, the line above is for `ranking`.